mockingbird/src/commonTest/kotlin/com/careem/mockingbird/test/FreezeStateVerification.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Careem, an Uber Technologies Inc. company
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.careem.mockingbird.test
+
+
+expect fun <T> T.verifyFreezeStateForMultiThread()
+
+expect fun <T> T.verifyFreezeStateForLocalThread()
```

Comment:
can we avoid expect here? we can define those functions directly in common

Reply:
I think in jvm it won't be frozen where needed 🤔